features/site_configuration.feature

```diff
@@ -161,8 +161,8 @@ Feature: Site configuration
       And I have a post layout that contains "Post Layout: {{ content }} built at {{ page.date | date_to_xmlschema }}"
       And I have an "index.html" page with layout "page" that contains "site index page"
       And I have a configuration file with:
-        | key         | value            |
-        | timezone    | America/New_York |
+        | key         | value        |
+        | timezone    | UTC+04:00    |
```

Comment:
> I believe we'd need to remove the timezone offsets in the date segment of the posts.

local tests show, there's no difference to whether the `post.date` is present or not..